app/internal/event.py

```diff
@@ -0,0 +1,11 @@
+import re
+
+from fastapi import HTTPException
+
+ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\t\n]+')
+
+
+def validate_zoom_link(location):
+    if not ZOOM_REGEX.findall(location):
```

Comment:
Maybe `ZOOM_REGEX.search... is not None` would fit here better (stops after 1 match)

Reply:
is None